pkg/v1/local/layer.go

```diff
@@ -0,0 +1,44 @@
+package local
```

Comment:
Is there a reason this shouldn't just go in the layout package? 🤔

Reply:
Not really, could go into layout package.